### ISSUE TITLE
inlineEdit panel and Inline chat/edit buttons don't support in VS Code

### ISSUE THREAD

**Issue by xielingdm** [continuedev/continue#2329]
### Validations

- [ ] I believe this is a way to improve. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [ ] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) that requests the same enhancement

### Problem

I see that JetBrains already supports inlineEdit panel and Inline chat/edit buttons as below
When will VScode be supported these features
![image](https://github.com/user-attachments/assets/f2371c49-fcfa-45cf-a82c-071599a2b69f)
<img width="665" alt="企业微信截图_5288bb61-5ccd-4ef2-8f73-d75230d373aa" src="https://github.com/user-attachments/assets/761bbc7b-d24f-4b39-ab0b-3387cebc20fc">

### Solution

Hopefully vscode will support it soon

**Comment by Patrick-Erichsen** [comment]
Hi @xielingdm , unfortunately VS Code doesn't allow extensions to render an inline chat panel, which is why we currently use the "[Quick Pick](https://code.visualstudio.com/api/ux-guidelines/quick-picks)" for inline edit.

**Comment by Patrick-Erichsen** [comment]
We don't support inline chat in VS Code since the underlying API is only available to GitHub Copilot. 

Valid feedback on the docs though, we're making some updates there soon and will keep this in mind 👍

**Comment by manusoftar** [comment]
> We don't support inline chat in VS Code since the underlying API is only available to GitHub Copilot.
> 
> Valid feedback on the docs though, we're making some updates there soon and will keep this in mind 👍

that is not true, Codeium is also doing inline editing from a long time so it is totally possible.

**Comment by d-msft** [comment]
I've noticed that other AI code extensions like Cody and double.bot have implemented this feature, it would be great to see it added to Continue as I find the sidebar clunky to use. Looking forward to any updates on this!

**Comment by Patrick-Erichsen** [comment]
<img width="627" height="199" alt="Image" src="https://github.com/user-attachments/assets/edf9bc89-c153-45ed-9779-c4d7b38c3a0e" />

@d-msft - I believe Cody lets you enter your input using the "Quick Pick" feature in VS Code. We used to have that but migrated to using the sidebar since it has more capabilities.

**Comment by d-msft** [comment]
@Patrick-Erichsen Yes that's it! Do you know which version still used the "Quick Pick" interface before switching to the sidebar? My use case is simple enough that the sidebar isn’t needed.

Update: I found out myself that the last release that used the "Quick Pick" interface is [v0.8.55-vscode](https://github.com/continuedev/continue/releases/tag/v0.8.55-vscode) and the last pre-release is [v0.9.224-vscode](https://github.com/continuedev/continue/releases/tag/v0.9.224-vscode).

**Comment by lephuongbg** [comment]
Have the situation regarding this changed since the Github Copilot Chat extension is now open source?

**Comment by Patrick-Erichsen** [comment]
Unfortunately no, the underlying APIs are still unavailable to third-party applications.
